Software/dist_windows/script32.iss

```diff
@@ -57,18 +57,13 @@ Source: "content/Qt5Network.dll"; DestDir: "{app}"; Flags: ignoreversion
 Source: "content/Qt5SerialPort.dll"; DestDir: "{app}"; Flags: ignoreversion
 Source: "content/Qt5Widgets.dll"; DestDir: "{app}"; Flags: ignoreversion
 Source: "gpl-3.0.txt"; DestDir: "{app}"; Flags: ignoreversion
-Source: "content/icudt54.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "content/icuin54.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "content/icuuc54.dll"; DestDir: "{app}"; Flags: ignoreversion
```

Comment:
I've tested it on a clean Windows 10 VM that I installed a few days ago. Got similar results:

![windows 10-2015-07-27-17-50-42](https://cloud.githubusercontent.com/assets/1467127/8910740/bbc378a0-3488-11e5-8d14-a6000f467426.png)

There is a couple of missing stuff though, but this is related to the system files itself. I'm not sure why, and Prismatik seems to start up fine.

Although I am encountering something odd. I have no clue why, but it should be displaying the Smoothness setting instead of those weird artifacts. Maybe it's because I've used a virtual device instead of a Lightpack?
![windows 10-2015-07-27-17-58-33](https://cloud.githubusercontent.com/assets/1467127/8910816/3cadeacc-3489-11e5-9eb3-07f0d642e4f4.png)